views/user/badges.php

```diff
@@ -42,5 +45,24 @@
                 </div>
             </div>
         <?php endforeach ?>
+
+        <h2>Forum Badges</h2>
+
+        <?php foreach($forumBadges as $badge): ?>
+        <div class="userbadge userbadge-forum">
+            <div class="userbadge-progress-bar" style="width: <?= round(0) ?>%"></div>
+            <div class="userbadge-info">
+                <h3>
+                    <?= Html::a(Html::encode($badge['name']), $badge['url']) ?>
+                    <?php if(isset($badge['grant_count'])): ?>
+                        <span class="x">x</span><span class="count"><?= $badge['grant_count'] ?></span>
+                    <?php endif ?>
+                </h3>
+                <p><?= $badge['description'] ?></p>
```

Comment:
Should be escaped against XSS.

Reply:
this is on purpose, description may contain links. They are managed by admins so users can not inject XSS here.

Reply:
OK.